x-pack/plugin/eql/src/main/java/org/elasticsearch/xpack/eql/parser/EqlParser.java

```diff
@@ -126,29 +127,6 @@ public void reportContextSensitivity(Parser recognizer, DFA dfa,
             this.ruleNames = ruleNames;
         }
 
-        @Override
```

Comment:
Without such a callback a correct identifier but starting with a digit, leads to a weird error as depending on the context, the parser might interpret the digits as a literal and the rest of the text as the actual identifier.

Reply:
I think to address this, we'll need to have another discussion about extending the language to support identifiers that don't match the form `[_A-Za-z][0-9A-Za-z_]*`.